pkg/migration/controllers/migration.go

```diff
@@ -2271,6 +2285,40 @@ func (m *MigrationController) applyResources(
 				unstructured.SetLabels(newLabels)
 			}
 
+			resource := &metav1.APIResource{
+				Name:       ruleset.Pluralize(strings.ToLower(objectType.GetKind())),
+				Namespaced: len(metadata.GetNamespace()) > 0,
+			}
+			var dynamicClient dynamic.ResourceInterface
+
+			stashCREnabled := false
+			gkv := o.GetObjectKind().GroupVersionKind()
+			keyName := fmt.Sprintf("%v-%v-%v", gkv.Group, gkv.Kind, gkv.Version)
+			if _, ok := appRegsStashMap[keyName]; ok {
+				stashCREnabled = appRegsStashMap[keyName]
+			}
+			if !stashCREnabled {
+				if resource.Namespaced {
+					dynamicClient = remoteClient.remoteInterface.Resource(
+						o.GetObjectKind().GroupVersionKind().GroupVersion().WithResource(resource.Name)).Namespace(metadata.GetNamespace())
+				} else {
+					dynamicClient = remoteClient.remoteAdminInterface.Resource(
+						o.GetObjectKind().GroupVersionKind().GroupVersion().WithResource(resource.Name))
+				}
+			} else {
+				log.MigrationLog(migration).Infof("getting configmap details for stashing resource %s/%s", unstructured.GetKind(), unstructured.GetName())
+				unstructured, err = getStashedConfigMap(unstructured)
+				if err != nil {
+					log.MigrationLog(migration).Warnf("unable to get stashed configmap content for object %s/%s, error: %v", unstructured.GetKind(), unstructured.GetName(), err)
```

Comment:
That function returns an error only in case of json marshal which is unlikely to fail. But we should continue in the for loop and avoid going ahead in this function.

Reply:
Instead of continue , addressed with a goto such that we can update the migration CR with error message.